docs/input/siemens-s7.md

```
@@ -30,9 +30,119 @@ input:
+| `DW` | Double Word | 4 bytes | `uint32` | No | `DB1.DW100` |
+| `DI` | Double Integer | 4 bytes | `int32` (signed) | No | `DB1.DI100` |
+| `R` | Real | 4 bytes | `float32` | No | `DB1.R200` |
+| `DT` | Date/Time | 8 bytes | `int64` (Unix nanoseconds) | No | `DB1.DT0` |
+| `S` | String | variable | `string` | Yes — max length (≥ 1) | `DB1.S30.10` |
+
+### The Extra Parameter
+
+Two data types require the extra parameter (the part after the second `.`):
+
+**Bit (`X`) — specify which bit (0–7) within the byte:**
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)